static/app/views/profiling/content.tsx

```diff
@@ -478,7 +479,8 @@ function ProfilingTransactionsContent(props: ProfilingTabContentProps) {
         <ProfilingOnboardingCTA />
       ) : (
         <Fragment>
-          {organization.features.includes('profiling-global-suspect-functions') ? (
+          {organization.features.includes('profiling-global-suspect-functions') &&
```

Comment:
nit: this would be a little cleaner if you swap the ordering to
```tsx
{organization.features.includes('continuous-profiling-compat') ? (
  ...
) : organization.features.includes('profiling-global-suspect-functions') ? (
  ...
) : (
  ...
)}
```